tests/unit/ordered-set-test.js

```diff
@@ -0,0 +1,22 @@
+import { module, test } from 'qunit';
+
+import { isEmpty } from '@ember/utils';
+
+import OrderedSet from 'ember-ordered-set';
```

Comment:
Are we going to use the scope?

Reply:
good point. will address in a follow-up PR that renames the project.